path_config_test.go

```diff
@@ -44,46 +44,92 @@ func setupLocalFiles(t *testing.T, b logical.Backend) func() {
 }
 
 func TestConfig_Read(t *testing.T) {
-	b, storage := getBackend(t)
-
-	cleanup := setupLocalFiles(t, b)
-	defer cleanup()
-
-	data := map[string]interface{}{
-		"pem_keys":               []string{testRSACert, testECCert},
-		"kubernetes_host":        "host",
-		"kubernetes_ca_cert":     testCACert,
-		"issuer":                 "",
-		"disable_iss_validation": false,
-		"disable_local_ca_jwt":   false,
-	}
-
-	req := &logical.Request{
-		Operation: logical.UpdateOperation,
-		Path:      configPath,
-		Storage:   storage,
-		Data:      data,
-	}
-
-	resp, err := b.HandleRequest(context.Background(), req)
-	if err != nil || (resp != nil && resp.IsError()) {
-		t.Fatalf("err:%s resp:%#v\n", err, resp)
+	type args struct {
+		data map[string]interface{}
 	}
-
-	req = &logical.Request{
-		Operation: logical.ReadOperation,
-		Path:      configPath,
-		Storage:   storage,
-		Data:      nil,
-	}
-
-	resp, err = b.HandleRequest(context.Background(), req)
-	if err != nil || (resp != nil && resp.IsError()) {
-		t.Fatalf("err:%s resp:%#v\n", err, resp)
+	tests := []struct {
+		name string
+		args args
+		want map[string]interface{}
+	}{
+		{
+			name: "token-review-jwt-is-unset",
+			args: args{
+				data: map[string]interface{}{
+					"pem_keys":               []string{testRSACert, testECCert},
+					"kubernetes_host":        "host",
+					"kubernetes_ca_cert":     testCACert,
+					"issuer":                 "",
+					"disable_iss_validation": false,
+					"disable_local_ca_jwt":   false,
+				},
+			},
+			want: map[string]interface{}{
+				"pem_keys":               []string{testRSACert, testECCert},
+				"kubernetes_host":        "host",
+				"kubernetes_ca_cert":     testCACert,
+				"issuer":                 "",
+				"disable_iss_validation": false,
+				"disable_local_ca_jwt":   false,
+				"token_reviewer_jwt_set": false,
+			},
+		},
+		{
+			name: "token-review-jwt-is-set",
+			args: args{
+				data: map[string]interface{}{
+					"pem_keys":               []string{testRSACert, testECCert},
+					"kubernetes_host":        "host",
+					"kubernetes_ca_cert":     testCACert,
+					"issuer":                 "",
+					"disable_iss_validation": false,
+					"disable_local_ca_jwt":   false,
+					"token_reviewer_jwt":     "test-token-review-jwt",
+				},
+			},
+			want: map[string]interface{}{
+				"pem_keys":               []string{testRSACert, testECCert},
+				"kubernetes_host":        "host",
+				"kubernetes_ca_cert":     testCACert,
+				"issuer":                 "",
+				"disable_iss_validation": false,
+				"disable_local_ca_jwt":   false,
+				"token_reviewer_jwt_set": true,
+			},
+		},
 	}
 
-	if !reflect.DeepEqual(resp.Data, data) {
-		t.Fatalf("Expected did not equal actual: expected %#v\n got %#v\n", data, resp.Data)
+	for _, tc := range tests {
+		b, storage := getBackend(t)
```

Comment:
It looks like the new test cases are not being run in their own test. Suggest running each test in `t.Run()`, then you can make all the test cases run in parallel.